### ISSUE TITLE
Support watching non-ts file everytime user make change

### ISSUE THREAD

**Issue by Dominic-Preap** [nestjs/nest-cli#581]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

I am not sure if it is possible but it would be nice if `@nestjs/cli` can watch and detect change on non-typescript file when define in assets.

## Expected behavior

For example if I have `get-data.sql` in `src` directory and when start nest, it will copy into `dist` but every time I make change in sql file, I have to start nest all over again.
```json
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "assets": ["**/*.sql"],
    "webpack": false
  }
}

```

## Environment

```
[System Information]
OS Version     : Windows 10
NodeJS Version : v12.14.1
YARN Version    : 1.22.0

[Nest CLI]
Nest CLI Version : 6.14.0

[Nest Platform Information]
platform-socket.io version : 6.10.14
platform-express version   : 6.10.14
microservices version      : 6.10.14
websockets version         : 6.10.14
passport version           : 6.1.1
schedule version           : 0.2.0
swagger version            : 4.1.12
typeorm version            : 6.2.0
common version             : 6.10.14
core version               : 6.10.14
jwt version                : 6.1.1

```

**Comment by kamilmysliwiec** [comment]
Would you like to create a PR for this issue?

**Comment by Dominic-Preap** [comment]
I'm taking a look into that.

**Comment by kamilmysliwiec** [comment]
Merged. Will be published as part of the next release (7.0.0)
